include/odp/api/spec/packet_io.h

```diff
@@ -489,6 +489,9 @@ typedef union odp_pktout_config_opt_t {
 		 */
 		uint64_t no_packet_refs  : 1;
 
+		/** Enable packet aging and drop */
+		uint64_t aging_ena  : 1;
```

Comment:
Like with tx_compl, this could mention that:
0: application will not request packet aging (default)
1: application will request packet aging

Reply:
Ack.